build-scripts/puddle-conf/errata-puddle-3.10-signed.el7.conf

```diff
@@ -0,0 +1,18 @@
+[puddle]
+type = simple
+rootdir = /mnt/rcm-guest/puddles/RHAOS
+mashroot = /mnt/rcm-guest/puddles/RHAOS/mash
+brewroot = file:///mnt/redhat/brewroot
+topurl = http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/
+name = AtomicOpenShift-signed
+version = 3.10
+emails = aos-team-art@redhat.com
+announcer = openshift-puddle <aos-team-art@redhat.com>
+publish = no
+external = /mnt/redhat/brewroot/repos/rhel-7-build/latest/$arch/
+arches = x86_64,ppc64le,aarch64,s390x
```

Comment:
Are **all** of these arches required for all of the 3.x `conf` additions here?

Reply:
Technically, no. Nothing is using aarch64, and 3.10 will certainly never use s390x. But I'd rather not have to go around updating these when we do add them in. What's the harm?